setup-buildengine.sh

```diff
@@ -46,8 +46,9 @@ OPT_UPLOAD_PATH=$DEF_UPLOAD_PATH
 OPT_COMPRESSION=9
 OPT_RELEASE="latest"
 OPT_TOOLING="Jolla-latest-Sailfish_SDK_Tooling-i486.tar.bz2"
-OPT_TARGET_ARM="Jolla-latest-Sailfish_SDK_Target-armv7hl.tar.bz2"
-OPT_TARGET_I486="Jolla-latest-Sailfish_SDK_Target-i486.tar.bz2"
+OPT_TARGET="Jolla-latest-Sailfish_SDK_Target-ARCH.tar.bz2"
```

Comment:
The variable name "OPT_TARGET" indicates that this could be overridden via options (perhaps --target=something) which seems to be not the case. Also, I'd like the variable name to indicate it more clearly that this is the filename template.

Reply:
Actually it is useless. It should be initialized as

```suggestion
OPT_TOOLING=
declare -A OPT_TARGETS=()
```

No OPT_TARGET_FILENAMES, no OPT_TARGET_ARCHES, no SAILFISH_DEFAULT_* is needed.

Reply:
So no mapping from Jolla-something to $OPT_TARGET_BASENAME-something needed?

Reply:
OPT_TOOLING and OPT_TARGET_* in the original code are initialized with the default file names, which never match the file names of images produced by imager nowadays, and so are never used.

Mapping based on SAILFISH_DEFAULT_* adds an unnecessary indirection and can be done more straightforward, as suggested in https://github.com/sailfishos/sdk-build-tools/pull/100#issuecomment-802594474:

```
installTooling "$OPT_TARGET_BASENAME-$OPT_RELEASE" "$OPT_TOOLING"
for arch in ${!OPT_TARGETS[*]} ; do
    installTarget "$OPT_TARGET_BASENAME-$OPT_RELEASE-$arch" "${OPT_TARGETS[$arch]}"
done
```

Reply:
Thanks that is what I needed to know, I was confused by that thinking it was still needed.

Reply:
hm, it's still there?